File: android/vbt-app/app/src/main/java/com/vbt/app/data/ble/BleConnectionState.kt
package com.vbt.app.data.ble

enum class BleConnectionState {
    DISCONNECTED,
    SCANNING,
    CONNECTING,
    CONNECTED,
    DISCONNECTING,
    // Niezamierzone rozłączenie - trwa automatyczna próba wznowienia połączenia
    RECONNECTING
}

File: ios/VBT/VBT/Core/BLE/BleConnectionState.swift
/// Port 1:1 z Android `data/ble/BleConnectionState.kt`.
enum BleConnectionState: Equatable {
    case disconnected
    case scanning
    case connecting
    case connected
    case disconnecting
    /// Niezamierzone rozłączenie — trwa automatyczna próba wznowienia połączenia.
    case reconnecting
}
